Makefile

```diff
@@ -19,6 +19,12 @@ clone: assert-dom0 ## Pulls the latest repo from work VM to dom0
 qubes-rpc: prep-salt ## Places default deny qubes-rpc policies for sd-app and sd-gpg
 	sudo qubesctl --show-output --targets sd-dom0-qvm-rpc state.highstate
 
+add-usb-autoattach: prep-salt ## Adds udev rules and scripts to sys-usb
+	sudo qubesctl --show-output --skip-dom0 --targets sys-usb state.highstate
```

Comment:
nit: On workstations built prior to this being merged, users will need to `make prep-dom0`, to copy the RPC policies in place. This is a one-time thing and not a huge deal, but calling prep-dom0 target before this here should resolve

Reply:
Will add that.